### DISCUSSION TITLE
新建一个版本时 自动重令名

### DISCUSSION THREAD

**Discussion by oxauwae** [Meloong-Git/PCL#514] [提问] [answered]
比如
1.18.2
再建一个的时候
1.18.2-1

**Comment by WForst-Breeze** [comment, marked as answer]
如有新功能建议，请提交[新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)qwq
